cli/context/docker/load.go

```diff
@@ -103,6 +103,7 @@ func (c *Endpoint) ClientOpts() ([]client.Opt, error) {
 			httpClient := &http.Client{
 				// No tls
 				// No proxy
+				Timeout: 30 * time.Second,
```

Comment:
🤔 Looks like this always cancels the connection, also for commands that may have to be running longer, for example;


```bash
time ./build/docker -H ssh://<remote machine ip> stats

...

CONTAINER ID   NAME                  CPU %     MEM USAGE / LIMIT     MEM %     NET I/O         BLOCK I/O         PIDS
4e67d3197705   nifty_hopper          0.00%     1.961MiB / 969.4MiB   0.20%     796B / 0B       8.19kB / 12.3kB   2
4aa507efd17b   relaxed_agnesi        0.00%     1.957MiB / 969.4MiB   0.20%     796B / 0B       0B / 12.3kB       2
5abb0f50e460   sweet_jang            0.00%     1.965MiB / 969.4MiB   0.20%     796B / 0B       8.19kB / 12.3kB   2
0f55fc6abd68   serene_elion          0.00%     1.953MiB / 969.4MiB   0.20%     796B / 0B       0B / 12.3kB       2
52e68d5c6261   stoic_wu              0.00%     1.957MiB / 969.4MiB   0.20%     796B / 0B       0B / 12.3kB       2
d268bc02bf6a   strange_ellis         0.00%     2.082MiB / 969.4MiB   0.21%     796B / 0B       164kB / 12.3kB    2
c7fd2fcd83eb   funny_albattani       0.00%     2.023MiB / 969.4MiB   0.21%     796B / 0B       73.7kB / 12.3kB   2
6521caf5828c   frosty_sammet         0.00%     2.148MiB / 969.4MiB   0.22%     796B / 0B       315kB / 12.3kB    2
ee9a3f87920c   sweet_feynman         0.00%     2.023MiB / 969.4MiB   0.21%     796B / 0B       77.8kB / 12.3kB   2
b1ecdb1b5eea   vigorous_meitner      0.00%     1.957MiB / 969.4MiB   0.20%     796B / 0B       24.6kB / 12.3kB   2
fd0f075961a3   stupefied_antonelli   0.00%     5.25MiB / 969.4MiB    0.54%     796B / 0B       5.2MB / 12.3kB    2
270a62d2d482   foo                   0.00%     3.16MiB / 969.4MiB    0.33%     170kB / 222kB   5.78MB / 12.3kB   2
context deadline exceeded (Client.Timeout or context cancellation while reading body)

________________________________________________________
Executed in   30.65 secs      fish           external
   usr time  204.56 millis    0.20 millis  204.36 millis
   sys time  192.99 millis    2.09 millis  190.90 millis
```

(same probably applies to pulling images which take longer than 30 seconds?)

Trying after the image was pulled initially works, but keeping the command running it produces an error after 30 seconds as well. While that error is printed, it's only printed once, but after that the shell still is functional 🤔

```bash
time ./build/docker -H ssh://<remote machine ip> run -it --rm alpine
/ # date
Fri Apr 28 14:30:33 UTC 2023
/ # ERRO[0031] error waiting for container:

/ # date
Fri Apr 28 14:31:05 UTC 2023
/ #
```

I also ran into this error, but only once, so not sure what caused it; in this case the image didn't exist locally, so wondering if something is because multiple API calls are made? (create  -> pull -> create -> attach?). I tried to reproduce it again, but wasn't able to, so perhaps it was just a networking glitch or something.


```bash
time ./build/docker -H ssh://<remote machine ip> run -it --rm alpine
Unable to find image 'alpine:latest' locally
latest: Pulling from library/alpine
f56be85fc22e: Already exists
Digest: sha256:124c7d2707904eea7431fffe91522a01e5a861a624ee31d03372cc1d138a3126
Status: Downloaded newer image for alpine:latest
command [ssh -- <remote machine ip> docker system dial-stdio] has exited with exit status 255, please make sure the URL is valid, and Docker 18.09 or later is installed on the remote host: stderr=kex_exchange_identification: Connection closed by remote host


________________________________________________________
Executed in    3.91 secs      fish           external
   usr time   51.93 millis    0.21 millis   51.72 millis
   sys time  104.69 millis    2.39 millis  102.30 millis
```

Reply:
^^^ just double-checked with the current CLI before this change, and that doesn't expose this issue